frontends/ol-util/src/components/CanonicalLink.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react"
```

Comment:
This file is moved from `frontends/open-discussions/src/components/CanonicalLink.js` and converted to tsx.